app/assets/javascripts/phrasing.js

```diff
@@ -150,6 +152,18 @@ var phrasing_setup = function(){
 
 };
 
+function disable_links() {
```

Comment:
please move this to the phrasing setup part because it's globally accessible this way.